docs/latest/tutorial/updates.md

```diff
@@ -1,11 +1,24 @@
 ---
-title: "Updating Applications"
+title: 'Updating Your Application'
 description: "There are several ways to update an Electron application. The easiest and officially supported one is taking advantage of the built-in Squirrel framework and Electron's autoUpdater module."
 slug: updates
 hide_title: false
 ---
 
-# Updating Applications
+:::info Tutorial parts
+This is part 7 of the Electron tutorial. The other parts are:
```

Comment:
I think again that the tutorial section should be be streamlined to support a single easy use-case. I think setting up `update.electronjs.org` updates might be a good use-case to extract into the tutorial, but advanced instructions should be in their own doc.